pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1042,7 +1042,16 @@ public TopicStats getStats(
             @ApiParam(value = "If return time of the earliest message in backlog")
             @QueryParam("getEarliestTimeInBacklog") @DefaultValue("false") boolean getEarliestTimeInBacklog) {
         validateTopicName(tenant, namespace, encodedTopic);
-        return internalGetStats(authoritative, getPreciseBacklog, subscriptionBacklogSize, getEarliestTimeInBacklog);
+        internalGetStatsAsync(authoritative, getPreciseBacklog, subscriptionBacklogSize, getEarliestTimeInBacklog)
```

Comment:
We need to use try/catch block to wrap `validateTopicName`.

Reply:
Good point, I suggest we should add an global exception hook to handle this.

Reply:
Hi @nodece , I have checked that `validateTopicName` already throw RestException, no need to use try/catch.